ui/imports/shared/popups/SendModal.qml

```diff
@@ -456,10 +460,10 @@ StatusDialog {
                                 }
                                 GasValidator {
                                     id: gasValidator
-                                    Layout.fillWidth: true
+                                    width: parent.width
                                     selectedAccount: popup.selectedAccount
                                     selectedAmount: amountToSendInput.text === "" ? 0.0 :
-                                                    parseFloat(amountToSendInput.text)
+                                                                                    parseFloat(amountToSendInput.text)
```

Comment:
This won't work, you have to parse according to the locale, e.g. https://doc.qt.io/qt-5/qml-qtqml-number.html#fromLocaleString-method

Reply:
(there are other occurences of this in the code)

Let's get this correct :)

Reply:
Will be fixed with https://github.com/status-im/status-desktop/pull/7258